packages/devtools-launchpad/src/components/LandingPage.css

```diff
@@ -93,6 +93,10 @@
   transition: var(--base-transition);
 }
 
+.landing-page .panel header input[type=button] {
+  flex: none;
+}
```

Comment:
Sorry to bother you with this, but instead of this change, could we instead rewrite the rule in l.85 from `.landing-page .panel header input` to `.landing-page .panel header input[type="search"]` ?
I prefer having a more specific rule for the search input and not having one for the button you added.